src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java

```diff
@@ -0,0 +1,232 @@
+public class TestTTLSteps extends FunctionalTestBase {
+        private static final Logger LOGGER = LoggerFactory.getLogger(TestTTLSteps.class);
+        private static final String BASE_URL = "localhost";
+        private static final String ENDPOINT = "/missed_notification";
+
+        private static final String SUBSCRIPTION_FILE_PATH = "src/functionaltests/resources/SubscriptionObject.json";
+        private static final String AGGREGATED_OBJECT_FILE_PATH = "src/functionaltests/resources/AggregatedObject.json";
+        private static final String MISSED_NOTIFICATION_FILE_PATH = "src/functionaltests/resources/MissedNotification.json";
+
+        private static String subscriptionObject;
+        private static String aggregatedObject;
+        private static String missedNotification;
+
+        private MockServerClient mockServerClient;
+        private ClientAndServer clientAndServer;
+
+        @Value("${missedNotificationDataBaseName}")
+        private String missedNotificationDatabase;
+
+        @Value("${missedNotificationCollectionName}")
+        private String missedNotificationCollectionName;
+
+        @Value("${notification.failAttempt}")
+        private String notificationFailAttempt;
+
+        @Value("${spring.data.mongodb.database}")
+        private String aggregationDataBaseName;
+
+        @Value("${aggregated.collection.name}")
+        private String aggregationCollectionName;
+
+        @Autowired
+        private MongoDBHandler mongoDBHandler;
+
+        @Autowired
+        private InformSubscription informSubscription;
+
+        @Before("@TestNotificationRetries")
+        public void beforeScenario() {
+                setUpMockServer();
+        }
+
+        @After("@TestNotificationRetries")
+        public void afterScenario() throws IOException {
+            LOGGER.debug("Shutting down mock servers.");
+            mockServerClient.close();
+            clientAndServer.stop();
+        }
+
+        // START TEST SCENARIOS
+
+        @Given("^Missed notification is created in database with index \"([A-Za-z0-9_]+)\"$")
+        public void missed_notification_is_created_in_database(
+            String indexName) throws IOException {
+            LOGGER.debug("Starting scenario @TestTTL");
+            missedNotification = FileUtils.readFileToString(new File(MISSED_NOTIFICATION_FILE_PATH), "utf-8");
+            BasicDBObject missedNotificationDocument = prepareDocumentWithIndex(missedNotification,
+                    indexName);
+
+            // setting 1 second TTL on index in db
+            mongoDBHandler.createTTLIndex(missedNotificationDatabase, missedNotificationCollectionName,
+                    indexName, 1);
+            Boolean isInserted = mongoDBHandler
+                    .insertDocument(missedNotificationDatabase,
+                            missedNotificationCollectionName,
+                            missedNotificationDocument.toString());
+            assertEquals(true, isInserted);
```

Comment:
`assertEquals(true, isInserted);`

A comment if this fails would be great, as it is you will only see that something expected true but got false.